lib/loaders/loaders_without_authentication/gravity.js

```diff
@@ -13,6 +13,7 @@ export default requestIDs => {
     fairsLoader: gravityLoader("fairs"),
     geneFamiliesLoader: gravityLoader("gene_families"),
     matchGeneLoader: gravityLoader("match/genes"),
+    geneLoader: gravityLoader(id => `/gene/${id}`),
```

Comment:
Maybe move this item up so that the list remains alphabetical?

Reply:
good call!